docs/usage.md

```diff
@@ -132,6 +132,10 @@ If you prefer, you can use [HISAT2](https://ccb.jhu.edu/software/hisat2/index.sh
 
 To use HISAT2, use the parameter `--aligner hisat2` or set `params.aligner = 'hisat2'` in your config file.
 
+When STAR has been chosen as the aligner then the pipeline sets the minimal lenght of an alignment to 15. This is to avoid some strange very short mappings that can come from the soft-clipping in STAR. If you which to change this value then you can modify it with the following flag:
+
+`--params.minAlgined`
```

Comment:
Just `--minAlgined`, not `--params.minAlgined`

Also, `Aligned` not `Algined`

We've used underscores for most other options, so maybe `--min_aligned`

Finally, I think it would be clearer to have something about the length, as min aligned sounds like it's the minimum number of aligned reads or something. So maybe `--min_aln_length`?

Oh, and if it's inline code then it's better to have it inline. If you're adding line breaks around it then it's better to use the triple backslashes.

Sorry 😅